src/transformers/models/funnel/modeling_tf_funnel.py

```diff
@@ -257,16 +257,17 @@ def get_position_embeds(self, seq_len, dtype=tf.float32, training=False):
                 # For block_index = 0 we only need the second one and leave the first one as None.
 
                 # First type
-                if block_index == 0:
-                    position_embeds_pooling = None
-                else:
+                position_embeds_pooling = None
```

Comment:
position_embeds_pooling seems to be only used at line 288. IMO it makes more sense to have a single if-else statement further below (at line 287):

```
if block_index != 0:
    position_embeds_pooling = tf.gather(pos_embed, rel_pos, axis=0) 
else:
    position_embeds_pooling = tf.fill(shape_list(position_embeds_no_pooling), value=-1.0)
```